xbmc/utils/CryptThreading.cpp

```diff
@@ -12,72 +12,6 @@
 
 #include <openssl/crypto.h>
 
-//! @todo Remove support for OpenSSL <1.0 in v19
-
-#define KODI_OPENSSL_NEEDS_LOCK_CALLBACK (OPENSSL_VERSION_NUMBER < 0x10100000L)
```

Comment:
`0x10100000L` == 1.1, so this has to be kept, if you only want to require 1.0.2 as minimum OpenSSL version.

Reply:
You are correct. I had to check the docs. For some reason I thought the conversation we had about 1.0.2 dealt with both callbacks. Thanks.